Plugins/FpeMonitoring/src/FpeMonitor.cpp

```diff
@@ -0,0 +1,344 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022-2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/FpeMonitoring/FpeMonitor.hpp"
+
+#include "Acts/Utilities/Helpers.hpp"
+
+#include <algorithm>
+#include <bitset>
+#include <cfenv>
+#include <csignal>
+#include <cstddef>
+#include <cstdint>
+#include <iostream>
+#include <iterator>
+#include <memory>
+#include <mutex>
+#include <stdexcept>
+#include <string_view>
+#include <vector>
+
+#include <boost/stacktrace/frame.hpp>
+#include <boost/stacktrace/safe_dump_to.hpp>
+#include <boost/stacktrace/stacktrace.hpp>
+#include <boost/stacktrace/stacktrace_fwd.hpp>
+#include <fenv.h>
+#include <signal.h>
```

Comment:
is `csignal` and `cfenv` not sufficient?

Reply:
Auto import from clangd. I'll check

Reply:
clang-format removes `cfenv` and `csignal` includes for some reason.

Reply:
`cfenv` and `csignal` are already included above no? so clang format will deduplicate them

Reply:
🤦